lighthouse-core/report/html/renderer/report-ui-features.js

```diff
@@ -447,7 +447,6 @@ class ReportUIFeatures {
    */
   getReportHtml() {
     this._resetUIState();
-    // @ts-ignore - technically documentElement can be null, but that's dumb - https://dom.spec.whatwg.org/#document-element
```

Comment:
this was unrelated, but noticed it when I was searching for `@ts-ignore`. Fixed in some tsc release since the comment was added.